config/src/main/java/org/springframework/security/config/web/server/ServerHttpSecurity.java

```diff
@@ -2677,6 +2701,42 @@ public HeaderSpec and() {
 
 		}
 
+		/**
+		 * Configures {@code Permissions-Policy} response header.
+		 *
+		 * @since 5.5
+		 * @see #permissionsPolicy(String)
+		 */
+		public final class PermissionsPolicySpec {
+
+			private PermissionsPolicySpec() {
+			}
+
+			private PermissionsPolicySpec(String permissionsPolicy) {
+				HeaderSpec.this.permissionsPolicy.setPolicy(permissionsPolicy);
+			}
+
+			/**
+			 * Sets the policy to be used in the response header.
+			 * @param permissionsPolicy a referrer policy
+			 * @return the {@link PermissionsPolicySpec} to continue configuring
+			 */
+			public PermissionsPolicySpec policy(String permissionsPolicy) {
```

Comment:
We should keep the parameter names consistent across this class and `PermissionsPolicyConfig`. We can name both of them either "permissionsPolicy" or "policy".